subsys/suit/storage/src/suit_storage_nrf54h20.c

```diff
@@ -88,11 +94,14 @@ struct suit_storage_nordic {
 			/* Application MPI Backup and digest. */
 			struct suit_storage_mpi_app app_mpi_bak;
 
+			/* Flags used for selecting the boot path. */
+			struct suit_storage_flags flags;
+
 			/* The last SUIT update report. */
-			uint8_t report_0[160];
+			uint8_t report_0[144];
```

Comment:
This seems unclear where does this number precisely come from... Perhaps it would be more readable to add the number of occupied bytes next to each field?
Like:
`struct suit_storage_flags flags; /* 16 bytes */